src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+import duke.command.*;
+import duke.exception.*;
+import duke.parser.*;
+import duke.ui.*;
+import duke.task.*;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Optional;
+import java.util.NoSuchElementException;
+
+
+public class Duke {
+    public static void main(String[] args) {
```

Comment:
Can we take out the logic for Running the bot into a separate method e.g  `run` and keep the `main` function as an entry point for the application?